tests/unit/get-page-schema.test.js

```diff
@@ -0,0 +1,95 @@
+import { getPageSchema } from '../../src/utils/get-page-schema';
+
+it('should correctly generate a schema for SEO on non-article pages', () => {
+    /*
+    Should have a schema looking like the following:
+    {{
+      '@context': 'http://schema.org',
+      '@type': 'Organization',
+      url: siteUrl,
+      logo: {
+        @type: 'imageObject',
+        url: logoURL
+      },
+    }}
+  */
+    const url = 'nonarticle.page';
+    const generatedSchema = JSON.parse(getPageSchema(url));
+    expect(generatedSchema['@context']).toBe('http://schema.org');
+    expect(generatedSchema['@type']).toBe('Organization');
+    expect(generatedSchema.url).toBe(url);
+    expect(generatedSchema.logo).toBeTruthy();
+    expect(generatedSchema.logo['@type']).toBe('imageObject');
+    expect(generatedSchema.logo.url).toBeTruthy();
+});
+
+it('should correctly generate a schema for SEO on article pages', () => {
+    /*
+  Should have a schema similar to the following:
+  {{'@context': 'https://schema.org',
+    '@type': 'BlogPosting',
+    url: articleURL,
+    headline: articleTitle,
+    description: aritcleDescription,
+    datePublished: datePublished,
+    datemodified: datemodified,
```

Comment:
Should this be camelCased? The example in the ticket was not.

Should dates be formatted in a specific way?

Reply:
I'd just follow the spec, but they'll be sending a confirmed markup soon hopefully

https://jsonld.com/blog-post/
https://developers.google.com/search/docs/guides/intro-structured-data